### ISSUE TITLE
 Additional property 'webviewInstallMode' is not allowed

### ISSUE THREAD

**Issue by Arelina** [tauri-apps/tauri#4574]
### Describe the bug

 Additional property 'webviewInstallMode' is not allowed

### Reproduction

<img width="466" alt="image" src="https://user-images.githubusercontent.com/42526528/177033200-0bd963a2-92fa-4529-8e52-386f0bff9fe7.png">

### Platform and versions

```shell
@tauri-apps/cli [NPM]: 1.0.2
  › @tauri-apps/api [NPM]: 1.0.1
  › tauri [RUST]: 1.0.2,
  › tauri-build [RUST]: 1.0.2,
  › tao [RUST]: 0.12.1,
  › wry [RUST]: 0.19.0,
```

**Comment by FabianLars** [comment]
Are you using the cargo cli? like `cargo tauri info`? If so what is its version? (`cargo tauri --version`)

And can you post the full error message please? It should either mean that your cli is outdated or the property is in the wrong place

Edit: Are you using `vue-cli-plugin-tauri`? the `tauri info` output doesn't look like you do, but i wanna be sure, because the plguin's cli version is pretty old right now.

**Comment by Arelina** [comment]
Yes, I use the vue-cli-plugin-tauri. Becase of I install @tauri-app/clis Manually  So It is show that version。

**Comment by FabianLars** [comment]
That explains it then, you can use the git version of the plugin by changing the dependency version in package.json to `github:tauri-apps/vue-cli-plugin-tauri` (run your package manager's install command afterwards0.

But there currently is a bug with the devserver, which is unreachable if started with `tauri:serve`

**Comment by lucasfernog** [comment]
Fixed in vue-cli-plugin-tauri 1.0.0.
